### ISSUE TITLE
White Artifacts in GIF

### ISSUE THREAD

**Issue by Noires** [bumptech/glide#1818]
**Glide Version**: 3.7.0 (Xamarin)

**Integration libraries**: None.

**Device/Android Version**: HP Pro Slate 12

**Issue details / Repro steps / Use case background**: Rarely the background of the gifs got some white artifacts. Sometimes the half of the background is white instead of transparent and sometimes everything is white with some transparent lines inbetween. So basically random. A example: 
![artifacts](https://cloud.githubusercontent.com/assets/4282056/24351434/ea3b833e-12e5-11e7-8ac0-ce959b9a5274.JPG)![artifacts2](https://cloud.githubusercontent.com/assets/4282056/24407912/98913bfe-13cc-11e7-9ff7-ae5bc81cfc89.JPG)

   `Glide.With(View.Context).Load(imageFile).SkipMemoryCache(true).DiskCacheStrategy(DiskCacheStrategy.None).DontTransform().DontAnimate().Into(ImageViewView);`

**Layout XML**:
Basically just a RelativeLayout where multiple imageviews are placed on.

The used gif:
![arrowleft](https://cloud.githubusercontent.com/assets/4282056/24351495/32023f64-12e6-11e7-912e-d08827717f16.gif)

**Comment by TWiStErRob** [comment]
Can you try with the `3.8.0-SNAPSHOT`? https://github.com/bumptech/glide/wiki/Snapshots

**Comment by Noires** [comment]
I tried it with the **glide-3.8.0-20160622.164216-13.jar** snapshot and it still appears.

**Comment by Noires** [comment]
I created the gif myself in gimp with the "combine" flag. Apparently if i use the "replace" flag in gimp, so that each layer gets replaced by the next for the animation, it still appears, BUT it just appears for one frame and then gets...well "replaced" again. 
Though iam not sure whats the cause yet.

**Comment by TWiStErRob** [comment]
I tried reproducing the issue with the [image you uploaded to GitHub](https://cloud.githubusercontent.com/assets/4282056/24351495/32023f64-12e6-11e7-912e-d08827717f16.gif), but didn't happen (Galaxy S4 API 23, Android Emu API 19). I did not use Xamarin though. Does it happen on other devices as well?
